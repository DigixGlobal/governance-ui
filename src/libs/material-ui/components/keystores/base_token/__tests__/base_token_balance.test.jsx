import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import BaseTokenBalance from '../base_token_balance';

jest.mock('../../../../helpers/stringUtils', () => ({
  parseBigNumber: () => 20,
}));

const web3 = {
  eth: {
    getBalance: jest.fn,
    balance: () => 10,
  },
};

describe('<BaseTokenBalance />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <BaseTokenBalance
        web3={web3}
        network={{}}
        address={{
          address: '0x34346f',
        }}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
