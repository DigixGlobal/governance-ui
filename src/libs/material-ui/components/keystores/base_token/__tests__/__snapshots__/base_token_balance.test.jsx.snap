// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseTokenBalance /> renders correctly 1`] = `
<BalanceLabel
  item={Object {}}
  value={20}
/>
`;
